content/android/getting-started.md

```diff
@@ -119,10 +114,37 @@ Also, you need to make sure your application has grant
 <uses-permission android:name="android.permission.INTERNET" />
 ```
 
-<a name="what's-next"></a>
+<a name="create"></a>
+## Creating a new project with configured SDK
+
+### Step 1: Installing Android Studio
+
+We recommend developing Android apps with Skygear Android SDK using
+[Android Studio](https://developer.android.com/studio/index.html).
+
+### Step 2: Downloading scaffolding project
+
+Clone the repo on github
+[SkygearIO/skygear-Scaffolding-Android](https://github.com/SkygearIO/skygear-Scaffolding-Android)
+or click
+[here](https://github.com/SkygearIO/skygear-Scaffolding-Android/archive/master.zip)
+to download the zip file. After you have done so, launch Android Studio,
+select **Open an existing Android Studio project** and find the scaffolding
+project you have just downloaded. Follow any recommendations Android Studio
+shows (such as installing the required SDKs) and then you are good to go.
+
+### We're done, Woohoo!
```

Comment:
We will need instructions with screenshot for showing an app running with Skygear configured